api/go.mod

```diff
@@ -14,10 +16,11 @@ require (
 	github.com/joho/godotenv v1.3.0
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/sergi/go-diff v1.2.0 // indirect
-	github.com/spf13/cobra v1.1.1
+	github.com/spf13/cobra v1.1.3
 	github.com/spf13/viper v1.7.0
 	github.com/stretchr/testify v1.6.1
-	github.com/tektoncd/pipeline v0.20.1-0.20210204110343-8c5a751b53ea
+	github.com/tektoncd/cli v0.18.0
```

Comment:
not sure if we should import cli dependency .. i think we didn't do this while fetching version
@vinamra28 ?

Reply:
right. This will add cyclic dependency problem

Reply:
Yep will update the change 👍🏻